filament/src/materials/dofDilate.mat

```diff
@@ -54,26 +54,32 @@ vec2 dilate(inout vec2 center, const vec2 tap) {
 }
 
 void postProcess(inout PostProcessInputs postProcess) {
-    // highp is not needed here, because we now the resolution is very low
-    vec2 uv = variable_vertex.xy;
-    vec2 size = vec2(textureSize(materialParams_tiles, 0));
-    ivec2 xy = ivec2(uv * size);
+    const float radius = 1.0;
+    // we need highp here just to maintain precision of 1/x
+    highp vec2 size = vec2(textureSize(materialParams_tiles, 0));
+    highp vec2 isize = 1.0 / size;
+
+    vec2 radii = float(radius) * isize;
```

Comment:
Highp?

Reply:
same thing here, we needed highp just for 1/x.